initializr-service/src/main/resources/application.yml

```diff
@@ -951,6 +958,67 @@ initializr:
           description: Hystrix circuit breaker on Pivotal Cloud Foundry
           groupId: io.pivotal.spring.cloud
           artifactId: spring-cloud-services-starter-circuit-breaker
+    - name: Azure
+      bom: azure
+      versionRange: 1.5.4.RELEASE
+      content:
+        - name: Azure Active Directory
+          id: active-directory
+          groupId: com.microsoft.azure
+          artifactId: azure-ad-integration-spring-boot-starter
+          description: Spring Security integration with Azure Active Directory for authentication
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-ad-integration-spring-boot-starter-sample
+              description: Sample code for using the starter
```

Comment:
Could you please replace that title to "Using Active Directory" or something like that so that it is consistent with other entries? This applies to all entries in this PR